Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -525,30 +525,59 @@ private bool IsLobbyNotificationValid(NotifyEventHandle handle)
         }
 
         // Helper method to keep the code cleaner
+
+        /// <summary>
+        /// Use to access functionality of [EOS_Lobby_AddNotifyLobbyUpdateReceived](https://dev.epicgames.com/docs/api-ref/functions/eos-lobby-add-notify-lobby-update-received)
+        /// The callback will only run if a listener is subscribed, which is done in <see cref="SubscribeToLobbyUpdates"/>.
+        /// </summary>
+        /// <param name="lobbyInterface">Handle to the lobby interface.</param>
+        /// <param name="notificationFn">Callback to receive notification when lobby member update is received</param>
```

Comment:
This should be "Callback to receive notification when lobby update is received." (looks like a copy - paste mistake). Also missing return value comment.

Reply:
Oh yea, this does have a return value. It's the intptr for where to find this notification for the purposes of subscription and unsubscription, right? I'll take a shot at communicating that, please review afterwards. `AchievementsInterface.AddNotifyAchievementsUnlocked` uses this line of text;

> handle representing the registered callback

So we'll use that.

`member` here is extra from copypaste, that text was removed.